packages/auth/src/api/index.test.ts

```diff
@@ -308,6 +309,66 @@ describe('api/_performApiRequest', () => {
     });
   });
 
+  context('referer policy exists on fetch request', () => {
+    afterEach(mockFetch.tearDown);
+
+    it('should have referrerPolicy set', async () => {
+      /* eslint-disable  no-var */
+      var referrerPolicySet: boolean = false;
+      /* eslint-enable no-var */
+      mockFetch.setUpWithOverride(
+        (input: RequestInfo | URL, request?: RequestInit) => {
+          if (request !== undefined && request.referrerPolicy !== undefined) {
+            referrerPolicySet = true;
+          }
+          return new Promise<never>((_, reject) =>
+            reject(new Error('network error'))
+          );
+        }
+      );
+      const promise = _performApiRequest(
+        auth,
+        HttpMethod.POST,
+        Endpoint.SIGN_UP,
+        request
+      );
+      await expect(promise).to.be.rejectedWith(
+        FirebaseError,
+        'auth/network-request-failed'
+      );
+      expect(referrerPolicySet).to.be.true;
+    });
+
+    it('should not have referrerPolicy set on Cloudflare workers', async () => {
+      sinon.stub(utils, 'isCloudflareWorker').returns(true);
+      /* eslint-disable  no-var */
+      var referrerPolicySet: boolean = false;
+      /* eslint-enable  no-var */
+      mockFetch.setUpWithOverride(
+        (input: RequestInfo | URL, request?: RequestInit) => {
+          if (request !== undefined && request.referrerPolicy !== undefined) {
+            referrerPolicySet = true;
+          }
+          return new Promise<never>((_, reject) =>
```

Comment:
optional: possible to resolve the promise instead of reject?

Reply:
Also curious why these tests have a rejected promise.

Reply:
Done! Updated tests so that these mocks resolve.